mixins/wallet-login.js

```diff
@@ -58,6 +73,37 @@ export default {
         return false;
       }
     },
+    async login(loginPayload = {}) {
```

Comment:
Instead of using `isLoginSuccessful` to record login result, just return a boolean in this function?